storage/innobase/ut/ut0dbg.cc

```diff
@@ -56,15 +55,6 @@ void ut_set_assert_callback(std::function<void()> &callback) {
 {
 #if !defined(UNIV_HOTBACKUP) && !defined(UNIV_NO_ERR_MSGS) && \
     !defined(XTRABACKUP)
-=======
-/** Report a failed assertion.
-@param[in] expr The failed assertion
-@param[in] file Source file containing the assertion
-@param[in] line Line number of the assertion */
-[[noreturn]] void ut_dbg_assertion_failed(const char *expr, const char *file,
-                                          ulint line) {
-#if !defined(UNIV_HOTBACKUP) && !defined(UNIV_NO_ERR_MSGS)
```

Comment:
I think these conflicts are related to oxygen style changes. Better to include them